qcodes/instrument_drivers/Aviosys/IP_Power_9258S.py

```diff
@@ -0,0 +1,92 @@
+from qcodes import Instrument, InstrumentChannel, ChannelList
+from qcodes.utils.validators import Enum
+from qcodes.utils.helpers import create_on_off_val_mapping
+import urllib
```

Comment:
@QCoDeS/core what do you think about the choice of `urllib` over [`requests`](http://docs.python-requests.org/en/master/)? Is it good to stick to the built-in in this case?